.travis.yml

```diff
@@ -26,10 +23,22 @@ cache:
     - $HOME/.gradle
     - $HOME/.m2/repository
 
-deploy:
-  provider: script
-  script: ./gradlew bintrayUpload
-  skip_cleanup: true
-  on:
-    branch: master
-    tags: true
+jobs:
+  include:
+    - stage: test
+      script: ./gradlew clean testDebugUnitTest jacocoTestReport
+    - stage: release
+      script: ./gradlew javadocRelease
+      deploy:
+        - provider: pages
+          skip_cleanup: true
+          github_token: $GITHUB_TOKEN # Set in travis-ci.org dashboard
+          local_dir: docs/
+          on:
+            all_branches: true
```

Comment:
Wouldn't we want to only do this on the master branch, @addisonElliott ?

Reply:
I spent an hour or two thinking the same thing. I used the same code that @flovilmart used when he added automated docs for JS and iOS. 

See here:
* https://github.com/parse-community/Parse-SDK-iOS-OSX/blob/master/.travis.yml#L38
* https://github.com/parse-community/Parse-SDK-iOS-OSX/blob/master/.travis.yml#L38

all_branches excludes PRs. It took me awhile to find documentation saying that. My guess is the thought process is that the only new branches on Parse-SDK-Android will be the master branch commit or new tags.